nodejs/week2/homework/warmup/index.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const app = express();
+app.get("/", (req, res) => res.send("nodejs week2 homework"));
```

Comment:
Make sure you check user inputs and return error in that case.

For the following input `http://172.22.236.48:3000/numbers/add` it is returned

`Addition of given numbers : NaN`